#include <stdlib.h>
#include "rpc_service.h"

namespace cobaya {

void RpcServiceImpl::GetAdminInfo(RpcController *ctl,
				  const MsgAdminReq *req,
				  MsgAdminRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::ControlSystem(RpcController *ctl,
				   const MsgMonitorReq *req,
				   MsgMonitorRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::GetClientInfo(RpcController *ctl,
				   const MsgClientReq *req,
				   MsgClientRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::AppendInfo(RpcController *ctl,
				const MsgDiscoveryReq *req,
				MsgDiscoveryRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::RegisterUser(RpcController *ctl,
				  const MsgRegUserReq *req,
				  MsgRegUserRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::LoginSystem(RpcController *ctl,
				 const MsgLoginReq *req,
				 MsgLoginRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::LogoutSystem(RpcController *ctl,
				  const MsgLogoutReq *req,
				  MsgLogoutRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::StartFlow(RpcController *ctl,
			       const MsgStartReq *req,
			       MsgStartRsp *rsp, Closure *done)
{

}

void RpcServiceImpl::StopFlow(RpcController *ctl,
			      const MsgStopReq *req,
			      MsgStopRsp *rsp, Closure *done)
{

}







} // namespace cobaya
